index.d.ts

```diff
@@ -0,0 +1,9 @@
+type ClassValue = string | number | ClassDictionary | ClassArray;
+
+interface ClassDictionary {
+	[id: string]: boolean;
+}
+
+interface ClassArray extends Array<ClassValue> { }
+
+export default function(...classes: ClassValue[]): string;
```

Comment:
@adidahiya There are many ways to do this...
`import classNames from 'classNames';`
`import * as classNames from 'classNames';`
`import classNames = require('classNames');`

Is there a "best" way? Is there a way to write the definition to work with all of the above?